### ISSUE TITLE
Why this happen on webpage? (showing source code) thanks!

### ISSUE THREAD

**Issue by francica** [sigurdga/django-jquery-file-upload#61]
Why this happen on webpage? (showing source code) thanks!
![1e0a663e0e3fd24](https://cloud.githubusercontent.com/assets/6868186/15726554/14be8fb2-2817-11e6-97f7-b3d89cc10576.jpg)

**Comment by sigurdga** [comment]
I see the same as you. I will try to get rid of the deprecation warnings from the newest django version before diving deep into this, as it may help to fix the causes for the warnings.

**Comment by francica** [comment]
thanks sigurdga, I tried windows, mac, windows with VMware to run the code. With the same install process, only VMware showed the correct html.  Hope this information helps.

**Comment by sigurdga** [comment]
After a while, I got it. Also upgraded the settings and other stuff to be compatible with Django 1.9. I tested all the different versions, and they uploads and has no console errors.

@francica: I guess it may have worked on your VMware because it was still on older Django.

On my long todo is to upgrade the blueimp library as well. Shouldn't be hard, but will take some time, so I'm not doing it now.
